openff/units/openmm.py

```diff
@@ -114,7 +114,8 @@ def _ast_eval(node):
         except AttributeError:
             raise MissingOpenMMUnitError(node.id)
         return b
-    # TODO: This toolkit code that had a hack to cover some edge behavior; not clear which tests trigger it
+    # TODO: This toolkit code that had a hack to cover some edge behavior
+    #       not clear which tests trigger it
     elif isinstance(node, ast.List):
         return ast.literal_eval(node)
```

Comment:
This can be tracked down to https://github.com/openforcefield/openff-toolkit/pull/173/commits/551a2d501b7d0e96d94abfa0153bfa5af4688f31 where I believe this was used to parse lists of numbers, which is no longer something this package does